kasa/cli.py

```diff
@@ -57,7 +57,7 @@
     "--type",
     envvar="KASA_TYPE",
     default=None,
-    type=click.Choice(TYPE_TO_CLASS, case_sensitive=False),
+    type=click.Choice(list(TYPE_TO_CLASS.keys()), case_sensitive=False),
```

Comment:
```suggestion
    type=click.Choice(list(TYPE_TO_CLASS), case_sensitive=False),
```
I'm not sure, but I think there's no need to call `keys()` to get the wanted effect. Would you mind testing if that works?